src/OSPSuite.Presentation/Presenters/Charts/ChartDisplayPresenter.cs

```diff
@@ -424,11 +430,13 @@ private void updateCurves()
          pruneCurves();
 
          Chart.Curves.Each(refreshCurveBinderFor);
+
+         _isLLOQVisible = _curveBinders.Any(x => x.LLOQ.HasValue && x.Curve.ShowLLOQ);
```

Comment:
I save this flag after each update. That way, no need to check for values if we know there are no LLOQ series either available or being shown